bot/constants.py

```diff
@@ -1,13 +1,9 @@
 """
-Loads bot configuration from environment variables
-and `.env` files. By default, this simply loads the
-default configuration defined thanks to the `default`
-keyword argument in each instance of the `Field` class
-If two files called `.env` and `.env.server` are found
-in the project directory, the values will be loaded
-from both of them, thus overlooking the predefined defaults.
-Any settings left out in the custom user configuration
-will default to the values passed to the `default` kwarg.
+Loads bot configuration from environment variables and `.env` files.
+
+By default, the values defined in the classes are used, these can be superceded by an env var with the same name.
```

Comment:
Should be `superseded` not `superceded`, although I think `overridden` sounds better anyway.
```suggestion
By default, the values defined in the classes are used, these can be overridden by an env var with the same name.
```